Sources/Plasma/Apps/plClient/winmain.cpp

```diff
@@ -1132,8 +1133,13 @@ int WINAPI WinMain(HINSTANCE hInst, HINSTANCE hPrevInst, LPSTR lpCmdLine, int nC
     // Set global handle
     gHInst = hInst;
 
-    CCmdParser cmdParser(s_cmdLineArgs, arrsize(s_cmdLineArgs));
-    cmdParser.Parse();
+    std::vector<plString> args(__argc);
+    for (size_t i = 0; i < __argc; i++) {
+        args.push_back(plString::FromWchar(__wargv[i]));
+    }
```

Comment:
This will create a vector of size `__argc * 2`, the first half of which is empty and the second half contains the args...  I believe what you want is an empty vector, followed by `args.reserve(__argc)`